app/editor/src/features/content/list-view/ContentListView.tsx

```diff
@@ -116,12 +121,12 @@ const ContentListView: React.FC = () => {
       try {
         if (!isLoading) {
           setIsLoading(true);
-          const data = await findContent(
-            makeFilter({
-              ...filter,
-            }),
-          );
-          const page = new Page(data.page - 1, data.quantity, data?.items, data.total);
+          const result = await findContentWithElasticsearch(getFilter(filter, actions), false);
+          const items = (result as any).hits?.hits?.map((h: { _source: IContentModel }) =>
```

Comment:
Result should not need to be `any`.  You can strongly type the `findContentWithElasticsearch` with `KnnSearchResponse`

Reply:
Updated, thanks